src/main.rs

```diff
@@ -391,15 +397,9 @@ fn run() -> Result<(), failure::Error> {
 
     let config_path = Path::new("./wrangler.toml");
 
-    if let Some(_matches) = matches.subcommand_matches("config") {
-        println!("Enter email: ");
-        let mut email: String = read!("{}\n");
```

Comment:
yah i think all the interactivity should stay up here.

Reply:
yup this is an easy fix! will do.